src/soilbiogeochem/SoilBiogeochemCompetitionMod.F90

```diff
@@ -293,7 +293,7 @@ subroutine SoilBiogeochemCompetition (bounds, num_soilc, filter_soilc,num_soilp,
 
       local_use_fun = use_fun
 
-      if (.not. use_nitrif_denitrif) then
+      if_nitrif: if (.not. use_nitrif_denitrif) then
```

Comment:
As above, I'm not used to seeing the description of what the logical statement is assuming written this way, but I like it. I also maybe don't pour over enough code :)

Reply:
I'm a big fan of labeling ifs and dos, particularly if they are closing spans of more than just a few lines. I like to have confidence in determining where the loops close when I'm looking over the code. Sometimes its hard to figure out which end is for which clause, and I don't trust comments.

Reply:
Yes labelling do and if statements means you can trust the start and end statements to be accurate, since the compiler is ensuring it. As Ryan points out you can't trust comments, as they can become wrong as the code changes. So I agree it's good to use these labels especially for long sections.